.env.production

```diff
@@ -0,0 +1 @@
+API_URL=
```

Comment:
The environment variable in development seems to be `REACT_APP_API_URL` while the production one is `API_URL`. Is this intended?

Reply:
Nope, it just my mistake. I already fix & push it.